decidim-assemblies/app/cells/decidim/assemblies/content_blocks/highlighted_assemblies_cell.rb

```diff
@@ -3,42 +3,34 @@
 module Decidim
   module Assemblies
     module ContentBlocks
-      class HighlightedAssembliesCell < Decidim::ViewModel
-        delegate :current_user, to: :controller
-
-        def show
-          render if highlighted_assemblies.any?
-        end
+      class HighlightedAssembliesCell < Decidim::ContentBlocks::HighlightedParticipatorySpacesCell
+        BLOCK_ID = "highlighted-assemblies"
 
-        def max_results
-          model.settings.max_results
-        end
+        delegate :current_user, to: :controller
 
-        def highlighted_assemblies
-          @highlighted_assemblies ||= OrganizationPrioritizedAssemblies
-                                      .new(current_organization, current_user)
-                                      .query
-                                      .with_attached_hero_image
-                                      .includes([:organization])
-                                      .limit(max_results)
+        def highlighted_spaces
+          @highlighted_spaces ||= OrganizationPrioritizedAssemblies
+                                  .new(current_organization, current_user)
+                                  .query
+                                  .with_attached_hero_image
+                                  .includes([:organization])
         end
 
         def i18n_scope
           "decidim.assemblies.pages.home.highlighted_assemblies"
         end
 
-        def decidim_assemblies
-          Decidim::Assemblies::Engine.routes.url_helpers
+        def all_path
+          Decidim::Assemblies::Engine.routes.url_helpers.assemblies_path
+        end
+
+        def max_results
+          model.settings.max_results
         end
 
         private
 
-        def cache_hash
-          hash = []
-          hash.push(I18n.locale)
-          hash.push(highlighted_assemblies.map(&:cache_key_with_version))
-          hash.join(Decidim.cache_key_separator)
-        end
+        def block_id = BLOCK_ID
```

Comment:
To keep consistency in the file, I would recommend removing the endless method definition. I would remove the `BLOCK_ID` constant too as it's only being used in this method.